etc/helm/test/global_test.go

```diff
@@ -5,13 +5,13 @@ package helmtest
 
 import (
 	"bytes"
-	"errors"
-	"fmt"
+	stderr "errors"
 	"io"
 	"reflect"
 	"testing"
 
 	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/pkg/errors"
```

Comment:
IMNSHO we should consider migrating away from `pkg/errors`, which has been archived for over two years, but … that is a story for a different time!

Reply:
@robert-uhl Our linter won't accept `fmt.Errorf` as a suitable wrapping, and `etc/` can't import `src/internal/errors`.  If we allowlist `fmt.Errorf` then lint fails with unnecessary directives in the few places in our codebase where we use it, and we don't want people that aren't familiar with the codebase using that one anyway.  So... it's complicated and annoying.  The thought process was to pick something that makes the linter happy and isn't a new dependency ;)

Moving `etc/helm` to `src/helm` is probably the right solution, but that breaks a lot of YAML files, so we can do that after the helm release is driven by Bazel.